src/EnergyPlus/InputProcessing/InputValidation.cc

```diff
@@ -100,10 +104,12 @@ bool Validation::validate(json const &parsed_input)
                     context += *it;
 
                 errors_.emplace_back(context + " - " + error.description);
-                if (max_context == 2 &&
-                    error.description ==
-                        "Object contains a property that could not be validated using 'properties' or 'additionalProperties' constraints: ''.") {
-                    errors_.emplace_back(context + " - Object name is required and cannot be blank or whitespace");
+                if (max_context == 2) {
+                    if (error.description == nameError) { // C++20: should mark this [[likely]]
```

Comment:
In this case, since we are doing string comparisons, and also this is just in the input processing part of the run, and also this is not expected to be a common occurrence, I'm not sure we'll need to add the `[[likely]]` declaration.  But in any case, the error handling change is good.